### DISCUSSION TITLE
I don't seem to have the ability to create new discussions

### DISCUSSION THREAD

**Discussion by taylorific** [polymathrobotics/caladan_examples#7] [Q&A] [answered]
Hi, while I see there's discussions in this repo. I don't see any button or ability to create new discussions. Do you know what I might be doing wrong?

<img width="1320" alt="2022-07-19_14-43-00" src="https://user-images.githubusercontent.com/57647141/179825537-84964d61-41ea-4788-ad1a-ade43b9566b2.png">

**Comment by amazing-bettyanne** [comment, marked as answer]
Hi @taylorific it looks like only people that have at least triage level permissions can create new discussions. When you do, you should see a button that looks something like below. This may be because our repo is still private. We're still learning how this discussion forum works. Let me circle back with GitHub and find out more about how discussions are used and how they work and I'll get back to you.

<img width="1506" alt="2022-07-19_14-53-51" src="https://user-images.githubusercontent.com/99734887/179827202-2c4e4f31-f1c9-4365-9e57-e53fac09e8a7.png">

**Reply by amazing-bettyanne** [reply]
I tried creating a public repo and limited the rights on a team member to "read", same issue, no "New discussion" button. So it's no different for public repos. Discussions don't work like we thought. Looking into this some more

<img width="1320" alt="2022-07-19_15-46-10" src="https://user-images.githubusercontent.com/99734887/179835593-178edf36-e65c-4cf4-b485-01b8ef897dbc.png">

**Reply by amazing-bettyanne** [reply]
I see something related on the community discussion board - trying to see if I can find that setting. https://github.com/github-community/community/discussions/29#discussioncomment-139055

**Reply by amazing-bettyanne** [reply]
Aha, I do see the setting in our org, and indeed it is disabled currently. OK @taylorific I tried enabling this setting, can you tell me if you see the "New discussion" button now?

<img width="829" alt="2022-07-19_16-03-59" src="https://user-images.githubusercontent.com/99734887/179838491-3f13c65b-a1c1-4e33-8ef9-1af6ea74a7e9.png">

**Reply by taylorific** [reply]
Yes, I see the button now - too scared to click on it yet as....pressure...but I see something now, so I think I'm good when I do feel the urge to create a discussion.

<img width="1320" alt="2022-07-19_16-07-27" src="https://user-images.githubusercontent.com/57647141/179839023-04285930-0296-4d57-8889-bd76749f851c.png">
